.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "@nuxtjs/eslint-config-typescript"
```

Comment:
I actually prefer using `.eslintrc.js` variant since it can be eslint-ed itself.

Also, I have been explicitly avoiding using `@nuxtjs/eslint-config-typescript` since it has some design issues (https://github.com/nuxt/eslint-config/issues/82).

Reply:
Actually `@nuxtjs/eslint-config-typescript` was used before too but it was scoped to .ts files to partially address the issues I have with it. If there is no good reason for this change then let's revert to the old approach.

Reply:
I'll revert that than

Reply:
I reverted the changes to the eslintrc and fixed the issues that came up after that